src/commands/initProjectForVSCode/InitVSCodeStep/BallerinaInitVSCodeStep.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { TaskDefinition } from 'vscode';
+import { func, hostStartCommand, hostStartTaskName, ProjectLanguage } from '../../../constants';
+import { getFuncWatchProblemMatcher } from '../../../vsCodeConfig/settings';
+import { convertToFunctionsTaskLabel } from '../../../vsCodeConfig/tasks';
+import { IJavaProjectWizardContext } from '../../createNewProject/javaSteps/IJavaProjectWizardContext';
+import { isBallerinaProject } from '../detectProjectLanguage';
+import { InitVSCodeStepBase } from './InitVSCodeStepBase';
+
+const ballerinaPackageTaskLabel: string = convertToFunctionsTaskLabel('package');
+
+export class BallerinaInitVSCodeStep extends InitVSCodeStepBase {
+    protected preDeployTask: string = ballerinaPackageTaskLabel;
+
+    private _debugSubpath: string = "target/azure_functions";
+
+    protected async executeCore(context: IJavaProjectWizardContext): Promise<void> {
```

Comment:
Is there a `getDebugConfiguration` for this?

Reply:
I'll look into this bit futher